crypto/verifier.go

```diff
@@ -31,8 +31,13 @@ import (
 	"github.com/google/trillian/crypto/sigpb"
 )
 
-// ErrVerify occurs whenever signature verification fails.
-var ErrVerify = errors.New("signature verification failed")
+var (
+	errVerify = errors.New("signature verification failed")
```

Comment:
You might want to make `errVerify` publicly exported so other modules can see what went wrong.